### PR TITLE
Explicitly specify the OS for GH trigger

### PR DESCRIPTION
**Pull request by gkhanna79**
skip ci please

### CODE REVIEW

**Comment by gkhanna79** [comment]
@mmitche PTAL

File: netci.groovy [hunk 1]
```diff
@@ -48,7 +49,7 @@ platformList.each { platform ->
 
             // Trigger a portable Linux build that runs on RHEL7.2
             buildCommand += " --portableLinux"
-            os = "RHEL7.2"
+            oosForGHTrigger = "PortableLinux"
```
Comment (mmitche) [inline netci.groovy]: oosForGHTrigger -> osForGHTrigger

**Comment by gkhanna79** [comment]
Fixed @mmitche. Aside from that, looks good?

**Comment by mmitche** [comment]
yep LGTM.

**Pull request merged**